pontoon/contributors/templates/contributors/profile.html

```diff
@@ -97,6 +104,17 @@ <h2 class="display-name">{{ contributor.first_name }}</h2>
                     <a class="button" href="{{ url('pontoon.contributors.settings') }}">Change Settings</a>
                 </div>
                 {% endif %}
+                {% set is_admin = (user.is_authenticated and (user.is_staff or user.is_superuser)) %}
```

Comment:
You can drop the `user.is_staff`- `user.is_superuser` is sufficient and used across the app.